src/static/index.html

```diff
@@ -24,14 +24,14 @@
     <link rel="stylesheet" type="text/css" href="https://igv.org/web/release/1.0.1/igv-1.0.1.css">
     <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
     <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.11.2/jquery-ui.min.js"></script>
-    <script type="text/javascript" src="https://igv.org/web/release/1.0.1/igv-1.0.1.min.js"></script>
+    <script type="text/javascript" src="https://igv.org/web/release/1.0.1/igv-1.0.1.js"></script>
```

Comment:
but the typeahead js is coming from assets which isn't a minified version, right?